docker-compose.yml

```diff
@@ -77,7 +77,7 @@ services:
     <<: *redis
 
   avalon: &avalon
-    image: avalonmediasystem/avalon:7.2.1-buster-dev-20210507
+    image: avalonmediasystem/avalon:7.2.1-buster-dev-20210517
```

Comment:
@apertome I don't see this tag in dockerhub. The newest one I see is `7.2.1-buster-dev-20210507`.

Reply:
Nice catch, @Dananji I pushed the image. Thanks.